pinot-broker/src/main/java/org/apache/pinot/broker/api/AccessControl.java

```diff
@@ -58,4 +58,19 @@ default boolean hasAccess(RequesterIdentity requesterIdentity) {
    * @return {@code true} if authorized, {@code false} otherwise
    */
   boolean hasAccess(RequesterIdentity requesterIdentity, Set<String> tables);
+
+  default boolean hasRBACAccess(RequesterIdentity requesterIdentity, String targetId,
+                                String targetType, String permission) {
```

Comment:
(minor) Let's follow [Pinot Style](https://docs.pinot.apache.org/developers/developers-and-contributors/code-setup#setup-ide)